src/App.css

```
@@ -1,3 +1,23 @@
 .App {
-  text-align: center;
+  width: 500px;
+  margin: auto;
+}
+
+.todo {
+  border: 2px solid gray;
+  margin-bottom: 10px;
+  padding: 5px;
+}
+
+p {
+  margin: 5px;
+}
+
+.errorMessage {
```

We have learned how to name classes.
```suggestion
.error_,message {
```